godot-core/src/builtin/aabb.rs

```diff
@@ -82,6 +82,21 @@ impl Aabb {
     */
 }
 
+impl std::fmt::Display for Aabb {
+    /// Formats `Aabb` to match godot's display style.
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// use godot::prelude::*;
+    /// let aabb = Aabb::new(Vector3::new(0.0, 0.0, 0.0), Vector3::new(1.0, 1.0, 1.0));
+    /// assert_eq!(format!("{}", aabb), "[P: (0, 0, 0), S: (1, 1, 1)]");
+    /// ```
```

Comment:
IIRC in other places we simply use
```suggestion
    /// Example:
    /// ```
    /// use godot::prelude::*;
    /// let aabb = Aabb::new(Vector3::new(0.0, 0.0, 0.0), Vector3::new(1.0, 1.0, 1.0));
    /// assert_eq!(format!("{}", aabb), "[P: (0, 0, 0), S: (1, 1, 1)]");
    /// ```
```
i.e. not a separate section. These function definitions are also rather short.